src/test_l1_dsDisplay.c

```diff
@@ -575,16 +562,20 @@ void test_l1_dsDisplay_negative_dsGetEDID(void) {
     // Step 02: Initialize the display sub-system
     result = dsDisplayInit();
     UT_LOG("\n In %s Return value: [%d]\n", __FUNCTION__, result);
-    UT_ASSERT_EQUAL(result, dsERR_NONE);
+    UT_ASSERT_EQUAL_FATAL(result, dsERR_NONE);
 
     // Step 03: Obtain a display device handle
-    for (int i = 0; i < sizeof(kSupportedPortTypes) / sizeof(kSupportedPortTypes[0]); i++) {
-        result = dsGetDisplay(kSupportedPortTypes[i], i, &displayHandle);
+    uint32_t numPorts = UT_KVP_PROFILE_GET_UINT32("dsDisplay/Number_of_ports");
```

Comment:
Please check the indentation for all the loops and update.. Check whole file

Reply:
done